app/controllers/returns/ExportedHumanMedicinesPlasticPackagingWeightController.scala

```diff
@@ -55,7 +55,7 @@ class HumanMedicinesPlasticPackagingWeightController @Inject()(
   def onPageLoad(mode: Mode): Action[AnyContent] =
     (identify andThen getData andThen requireData) {
       implicit request =>
-        val preparedForm = request.userAnswers.fill(HumanMedicinesPlasticPackagingWeightPage, form)
+        val preparedForm = request.userAnswers.fill(ExportedHumanMedicinesPlasticPackagingWeightPage, form)
 
         request.userAnswers.get[TaxReturnObligation](ObligationCacheable) match {
```

Comment:
Do we need TaxReturnObligation here? It seem this is not used in the page/view, so it can be removed

Reply:
I think this can now be removed from all pages. As its outside of the scope of this ticket I suggest we raise a ticket to tidy all pages?

Reply:
ok